src/global/mixins/mixins_card.scss

```diff
@@ -45,6 +47,10 @@
     }
   }
 
+  &:active {
+    outline: var(--card-outline);
+  }
+
```

Comment:
--card-outline is never defined? and an :active state does not exist on a card. it probably would have to be set by class or property...

maybe just remove it.